src/main/java/mate/academy/spring/service/UserServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.spring.service;
+
+import java.util.List;
+import mate.academy.spring.dao.UserDao;
+import mate.academy.spring.model.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+    private UserDao userDao;
+
+    @Override
+    public void add(User user) {
+        userDao.add(user);
+    }
+
+    @Override
+    public List<User> getAll() {
+        return userDao.getAll();
+    }
+
+    @Autowired
+    private void setUserDao(UserDao userDao) {
+        this.userDao = userDao;
+    }
```

Comment:
let's use uniform style of setting dependencies
better use constructor injection everywhere

Reply:
I've just liked that getter can be private for better incapsulation